src/tests/stdlib/change_owner.ab

```diff
@@ -7,10 +7,10 @@ main {
     $touch {tmpdir}/amber-symbolic$ failed {
         echo "It wasn't possible to create {tmpdir}/amber-symbolic"
     }
-    if change_owner(unsafe $whoami$, "{tmpdir}/amber-symbolic") {
+    if change_owner(trust $whoami$, "{tmpdir}/amber-symbolic") {
         echo "Succeded"
     } else {
         echo "File {tmpdir}/amber-symbolic not exists"
     }
-    unsafe $rm -fr {tmpdir}$
+    trust $rm {tmpdir}$
```

Comment:
I'm shocked that this diff is shown. I didn't touch this code. This could be some old code that I must have committed from other branch or something.

This particular example won't work because it's a directory. The `-r` flag is necessary at the very least. I'll fix these later today.

Reply:
I resolved a merge conflict in this PR as we merged a PR that fixed this test and others.

Reply:
I see